utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -12,60 +12,115 @@
 import platform
 
 
-class StdlibDeploymentTarget(object):
+class Platform(object):
+    """
+    Abstract representation of a platform Swift can run on.
+    """
+
+    def __init__(self, name, archs, sdk_name=None, is_darwin=False):
+        """
+        Create a platform with the given name and list of architectures.
+        """
+        self.name = name
+        self.targets = [Target(self, arch) for arch in archs]
```

Comment:
I would prefer to keep these as allArchs or `architectures` or something. The fact that it's a target is already in the name; if the general pattern is `StdlibDeploymentTarget.{platform}.{architecture}`, `StdlibDeploymentTarget.{platform}.targets` isn't very clear that it's an aggregate of all architectures.